corehq/apps/app_manager/static/app_manager/json/commcare-app-settings.yaml

```diff
@@ -173,3 +173,11 @@
   values: ['none', '0', '1', '2', '4', '6', '8', '12', '24']
   value_names: ['Use project level setting', 'Update on every sync', '1 hour', '2 hours', '4 hours', '6 hours', '8 hours', '12 hours', '24 hours']
   default: 'none'
+
+- id: mobile_ucr_restore_version
+  name: Mobile UCR Restore Version
+  description: Version of mobile UCRs to use.
```

Comment:
Link to docs with more info would be handy for future

Reply:
Writing up some docs for GS/enikshay team tomorrow about this. Will link once those are complete

Reply:
linked confluence page https://github.com/dimagi/commcare-hq/pull/18273/commits/ddfcff2dff97931b267df32064db62042cd48ebb